crates/assert-instr-macro/src/lib.rs

```diff
@@ -49,7 +49,7 @@ pub fn assert_instr(
         .replace(':', "_")
         .replace(char::is_whitespace, "");
     let assert_name = syn::Ident::new(&format!("assert_{}_{}", name, instr_str), name.span());
-    let shim_name = syn::Ident::new(&format!("{}_shim_{}", name, instr_str), name.span());
+    let shim_name = syn::Ident::new(&format!("stdsimd_test_shim_{}_{}", name, instr_str), name.span());
```

Comment:
turns out `shim` is a really common name, and I was getting collisions against a `vtable.shim` inside `std::once::Once`... this disambiguates that.

TODO: add a comment here, or refactor this.